diff --git a/framework/modules/CMakeLists.txt b/framework/modules/CMakeLists.txt
index 9a32fa2..4e5df45 100644
--- a/framework/modules/CMakeLists.txt
+++ b/framework/modules/CMakeLists.txt
@@ -12,9 +12,6 @@ PRIVATE
     ${CMAKE_CURRENT_SOURCE_DIR}/saf_sh/saf_sh_internal.c
     ${CMAKE_CURRENT_SOURCE_DIR}/saf_sh/saf_sh_internal.h
     ${CMAKE_CURRENT_SOURCE_DIR}/saf_sh/saf_sh.c
-    ${CMAKE_CURRENT_SOURCE_DIR}/saf_tracker/saf_tracker_internal.c
-    ${CMAKE_CURRENT_SOURCE_DIR}/saf_tracker/saf_tracker_internal.h
-    ${CMAKE_CURRENT_SOURCE_DIR}/saf_tracker/saf_tracker.c
     ${CMAKE_CURRENT_SOURCE_DIR}/saf_utilities/saf_utility_bessel.c
     ${CMAKE_CURRENT_SOURCE_DIR}/saf_utilities/saf_utility_complex.c
     ${CMAKE_CURRENT_SOURCE_DIR}/saf_utilities/saf_utility_decor.c
diff --git a/test/CMakeLists.txt b/test/CMakeLists.txt
index 16f7495..f4e3d5f 100644
--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -11,12 +11,10 @@ PRIVATE
     ${CMAKE_CURRENT_SOURCE_DIR}/src/test__vbap_module.c
     ${CMAKE_CURRENT_SOURCE_DIR}/src/test__cdf4sap_module.c
     ${CMAKE_CURRENT_SOURCE_DIR}/src/test__examples.c
-    ${CMAKE_CURRENT_SOURCE_DIR}/src/test__hades_module.c
     ${CMAKE_CURRENT_SOURCE_DIR}/src/test__hoa_module.c
     ${CMAKE_CURRENT_SOURCE_DIR}/src/test__hrir_module.c
     ${CMAKE_CURRENT_SOURCE_DIR}/src/test__resources.c
     ${CMAKE_CURRENT_SOURCE_DIR}/src/test__reverb_module.c
     ${CMAKE_CURRENT_SOURCE_DIR}/src/test__sh_module.c
     ${CMAKE_CURRENT_SOURCE_DIR}/src/test__sofa_reader_module.c
-    ${CMAKE_CURRENT_SOURCE_DIR}/src/test__tracker_module.c
     ${CMAKE_CURRENT_SOURCE_DIR}/src/test__utilities_module.c
 )
